app/main.py

```diff
@@ -1 +1,51 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
```

Comment:
The `skills` attribute is defined as a class variable, which means it will be shared across all instances of `SoftwareEngineer`. This can lead to unexpected behavior if you modify `skills` for one instance, as it will affect all instances. Consider defining `skills` as an instance variable within the constructor (`__init__` method) instead.